### ISSUE TITLE
Update Plupload

### ISSUE THREAD

**Issue by adamjohnson** [wvuweb/cleanslate-cms#156]
We periodically get reports of people having trouble uploading PDF's (like [this thousand+ page PDF](https://drive.google.com/file/d/0B4gR5PUC4p7wUFctZmQ5MF8yYzQ/view?usp=sharing)). We're not sure the reason for this but perhaps updating [Plupload](http://www.plupload.com/) would help. Right now we use version `1.0.6` from January 31, 2012. [Plupload-rails's](https://github.com/gucki/plupload-rails) most recent version is `1.2.1` released on February 8th, 2014. It looks like the Rails 4 branch uses Plupload-rails version `1.1.0`.

@nreckart Do you think updating Plupload to the most recent version is feasible?

**Comment by nreckart** [comment]
I don't think updating Plupload would fix the PDF uploading issue, but keeping things up-to-date is usually a good idea.

When you upload a PDF, the system generates a preview image for it to be displayed in the admin UI. I think the uploading issue is caused by this process, which probably pukes for huge PDFs. After doing a little research, I'm not sure there's an efficient way to actually generate the preview image. We may need to just nix PDF preview images and go with a basic icon like we do for other non image files.

**Comment by adamjohnson** [comment]
Works for me. Usually the PDF preview is not very helpful anyway (small text in a PDF).

Tally ho.

**Comment by zeroedin** [comment]
Dunno if relavant, but any chance you are trying to create a preview icon for all pages in the PDF?

http://stackoverflow.com/questions/467793/how-do-i-convert-a-pdf-document-to-a-preview-image-in-php

PHP example and again not sure if its at all helpful.

**Comment by nreckart** [comment]
No, it takes the first image, but I'm not sure exactly how ImageMagick and GhostScript interact. Limited testing shows that GS borks on large PDFs and it has actually killed my test VM 3 times. 

The previews were a nice gimmicky feature, but as @adamjohnson pointed out, they aren't all that useful. I can cut them out and fix the issue while speeding up PDF uploads in general. Any objections?

**Comment by zeroedin** [comment]
Sounds good to me.

**Comment by dustinmazon** [comment]
No preview is definitely ok. +1 @adamjohnson It's too small to see anyway.

**Comment by nreckart** [comment]
Should be able to upload gigantic PDFs now. The actual issue of updating Plupload is still open.
